Automattic/wp-calypso
# Use visual regression testing

#### Changes proposed in this Pull Request

* This PR implements Visual regression testing framework, initial tests, and CI scheduling.
The tests are scheduled to run daily. Reporting and notification will be added in future PR.

#### Testing instructions



* Check out and run `yarn test-visual` from the root. A report will be generated with changes that differ from the reference images. To replace the reference images, run `yarn test-visual-reference`



Related to #51916
